mps-model-adapters/src/main/kotlin/org/modelix/model/mpsadapters/MPSLanguageRepository.kt

```diff
@@ -21,11 +21,25 @@ import jetbrains.mps.smodel.adapter.structure.MetaAdapterFactory
 import jetbrains.mps.smodel.language.ConceptRegistry
 import jetbrains.mps.smodel.language.LanguageRegistry
 import jetbrains.mps.smodel.runtime.illegal.IllegalConceptDescriptor
+import org.jetbrains.mps.openapi.language.SAbstractConcept
 import org.jetbrains.mps.openapi.module.SRepository
 import org.modelix.model.api.IConcept
 import org.modelix.model.api.ILanguageRepository
 
-class MPSLanguageRepository(private val repository: SRepository) : ILanguageRepository {
+class MPSLanguageRepository(val repository: SRepository) : ILanguageRepository {
+
+    private val concepts = mutableMapOf<String, MPSConcept>()
```

Comment:
I'm not sure, if we can make this memory trade-off.
Since we have projects with a lot of concepts, it may not be feasible to cache them all.
@languitar what do you think?

Reply:
I have no clue how much memory all concepts would take up in our use case. We'd have to test that. But they are probably loaded somewhere in any case?

Reply:
@mhuster23 Can you give me such example projects? I could do some measurements to see if caching is a real issue in that case.

Reply:
We probably can't give you access to the project. I just counted and found 6500 concepts

Reply:
I don't understand why this cache was introduced. It's not an expensive computation and the only usage I can find already does some caching.

Reply:
In [5c08a24](https://github.com/modelix/modelix.core/pull/237/commits/5c08a244b6c5bacb28a89d0a24716589a4d8caf2), I removed the cache so this conversation became outdated.